code/drasil-example/projectile/lib/Drasil/Projectile/Requirements.hs

```diff
@@ -39,8 +39,13 @@ calcValuesDesc = foldlSent [S "Calculate the following" +: plural value,
     ch offset    +:+ fromSource offsetIM,
     ch message   +:+ fromSource messageIM
   ]]
-outputValuesDesc = foldlSent [atStart output_, ch message,
-  fromSource messageIM `S.and_` ch offset, fromSource offsetIM]
+outputValuesDesc = atStart output_ +:+. outputs
+  where
+    outputs = foldlList Comma List $ map foldlSent_ [ 
+        [ch flightDur, fromSource messageIM],
```

Comment:
`flightDur` doesn't come from `messageIM`

Reply:
:facepalm:  Thank you. I spent too much time fiddling with the sentence combinators that I rushed.